ga4gh/datamodel/rna_quantification.py

```diff
@@ -0,0 +1,390 @@
+                            expressionId=None, featureGroupId=None,
+                            threshold=0.0):
+        """
+        Returns the list of ExpressionLevels in this RNA Quantification.
+        """
+        if expressionId is not None:
+            rawId = datamodel.ExpressionLevelCompoundId.deobfuscate(
+                expressionId)
+            expressionId = string.split(rawId, ":")[-1]
+        with self._db as dataSource:
+            expressionsReturned = dataSource.searchExpressionLevelsInDb(
+                rnaQuantID, pageToken=pageToken,
+                pageSize=pageSize, expressionId=expressionId,
+                featureGroupId=featureGroupId, threshold=threshold)
+        return [ExpressionLevel(self, expressionEntry) for
+                expressionEntry in expressionsReturned]
+
+    def getExpressionLevel(self, compoundId):
+        rawId = datamodel.ExpressionLevelCompoundId.deobfuscate(compoundId)
+        localExpressionId = string.split(rawId, ":")[-1]
+        with self._db as dataSource:
+            expressionReturned = dataSource.getExpressionLevelById(
+                localExpressionId)
+
+        if expressionReturned is not None:
+            return ExpressionLevel(self, expressionReturned)
+        else:
+            raise exceptions.ExpressionLevelNotFoundException(compoundId)
+
+    def getFeatureGroups(self, rnaQuantID, pageToken=0, pageSize=None,
+                         featureGroupId=None):
+        """
+        Returns the list of FeatureGroups in this RNA Quantification.
+
+        for now the feature group data is autogenerated by examining the
+        relevant expression data file
+        """
+        if featureGroupId is not None:
+            rawId = datamodel.FeatureGroupCompoundId.deobfuscate(
+                featureGroupId)
+            featureGroupId = string.split(rawId, ":")[-1]
+        with self._db as dataSource:
+            featureGroupsReturned = dataSource.searchFeatureGroupsInDb(
+                rnaQuantID, pageToken=pageToken,
+                pageSize=pageSize, featureGroupId=featureGroupId)
+        return [FeatureGroup(self, featureGroupEntry) for
+                featureGroupEntry in featureGroupsReturned]
+
+    def getFeatureGroup(self, compoundId):
+        """
+        for now the feature group data is autogenerated by examining the
+        relevant expression data file
+        """
+        rawId = datamodel.FeatureGroupCompoundId.deobfuscate(compoundId)
+        localFeatureGroupId = string.split(rawId, ":")[-1]
+        with self._db as dataSource:
+            featureGroupReturned = dataSource.getFeatureGroupById(
+                localFeatureGroupId)
+
+        if featureGroupReturned is not None:
+            return FeatureGroup(self, featureGroupReturned)
+        else:
+            raise exceptions.FeatureGroupNotFoundException(compoundId)
+
+
+_rnaQuantColumns = [('id', 'TEXT'),
```

Comment:
Pleas use hanging indents for these --- planned snake_case changes will break the alignment.